{"timestamp":"2025-09-05T05:25:37.188293","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-05T05:25:37.188761","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/airbyte-and-dbt-with-airflow.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-05T05:25:37.217038","level":"info","event":"Trying to parse the dbt project using dbt ls cache cosmos_cache__airbyte_dynamic_connection__my_project_dbt_group_11...","logger":"cosmos.dbt.graph"}
{"timestamp":"2025-09-05T05:25:37.258032","level":"info","event":"Cosmos performance: time to calculate cache identifier cosmos_cache__airbyte_dynamic_connection__my_project_dbt_group_11 for current version: 0.03224320901790634","logger":"cosmos.cache"}
{"timestamp":"2025-09-05T05:25:37.258148","level":"info","event":"Cosmos performance [5f5c1e41ce68|135]: The cache size for cosmos_cache__airbyte_dynamic_connection__my_project_dbt_group_11 is 4337","logger":"cosmos.dbt.graph"}
{"timestamp":"2025-09-05T05:25:37.258433","level":"info","event":"Cosmos performance: Cache hit for cosmos_cache__airbyte_dynamic_connection__my_project_dbt_group_11 - 2499a7507d070b7bbbbe009f3f50a3a8,d41d8cd98f00b204e9800998ecf8427e","logger":"cosmos.dbt.graph"}
{"timestamp":"2025-09-05T05:25:37.258810","level":"info","event":"Total nodes: 6","logger":"cosmos.dbt.graph"}
{"timestamp":"2025-09-05T05:25:37.258863","level":"info","event":"Total filtered nodes: 6","logger":"cosmos.dbt.graph"}
{"timestamp":"2025-09-05T05:25:37.258927","level":"info","event":"Cosmos performance (airbyte_dynamic_connection__my_project_dbt_group_11) -  [5f5c1e41ce68|135]: It took 0.0422s to parse the dbt project for DAG using LoadMode.DBT_LS_CACHE","logger":"cosmos.converter"}
{"timestamp":"2025-09-05T05:25:37.259271","level":"info","event":"Project /opt/airflow/dbt/my_project_dbt does not have {'packages.yml', 'dependencies.yml'}","logger":"cosmos.dbt.project"}
{"timestamp":"2025-09-05T05:25:37.260072","level":"info","event":"Project /opt/airflow/dbt/my_project_dbt does not have {'packages.yml', 'dependencies.yml'}","logger":"cosmos.dbt.project"}
{"timestamp":"2025-09-05T05:25:37.260984","level":"info","event":"Project /opt/airflow/dbt/my_project_dbt does not have {'packages.yml', 'dependencies.yml'}","logger":"cosmos.dbt.project"}
{"timestamp":"2025-09-05T05:25:37.261467","level":"info","event":"Project /opt/airflow/dbt/my_project_dbt does not have {'packages.yml', 'dependencies.yml'}","logger":"cosmos.dbt.project"}
{"timestamp":"2025-09-05T05:25:37.261960","level":"info","event":"Project /opt/airflow/dbt/my_project_dbt does not have {'packages.yml', 'dependencies.yml'}","logger":"cosmos.dbt.project"}
{"timestamp":"2025-09-05T05:25:37.262391","level":"info","event":"Project /opt/airflow/dbt/my_project_dbt does not have {'packages.yml', 'dependencies.yml'}","logger":"cosmos.dbt.project"}
{"timestamp":"2025-09-05T05:25:37.262997","level":"info","event":"Cosmos performance (airbyte_dynamic_connection__my_project_dbt_group_11) - [5f5c1e41ce68|135]: It took 0.00409s to build the Airflow DAG.","logger":"cosmos.converter"}
{"timestamp":"2025-09-05T05:25:37.264537","level":"warning","event":"/home/airflow/.local/lib/python3.12/site-packages/pydantic/main.py:463: UserWarning: Pydantic serializer warnings:\n  PydanticSerializationUnexpectedValue(Expected `str` - serialized value may not be as expected [input_value={'version': '2499a7507d07...y_project_dbt_group_11'}, input_type=dict])\n  return self.__pydantic_serializer__.to_python(\n","logger":"py.warnings"}
{"timestamp":"2025-09-05T05:25:37.296289Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.296468Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.297626Z","level":"info","event":"Current task name:create_custom_source_and_discover_schema","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.297803Z","level":"info","event":"Dag name:airbyte_dynamic_connection","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.323752","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"ConnectionError","exc_value":"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":918,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1213,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/airbyte-and-dbt-with-airflow.py","lineno":53,"name":"create_custom_source_and_discover_schema"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/api.py","lineno":115,"name":"post"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/api.py","lineno":59,"name":"request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/sessions.py","lineno":589,"name":"request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/sessions.py","lineno":703,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/adapters.py","lineno":682,"name":"send"}],"is_group":false,"exceptions":[]},{"exc_type":"ProtocolError","exc_value":"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/requests/adapters.py","lineno":667,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":841,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/util/retry.py","lineno":474,"name":"increment"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/util/util.py","lineno":38,"name":"reraise"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":787,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":534,"name":"_make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connection.py","lineno":565,"name":"getresponse"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":1430,"name":"getresponse"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":331,"name":"begin"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":300,"name":"_read_status"}],"is_group":false,"exceptions":[]},{"exc_type":"RemoteDisconnected","exc_value":"Remote end closed connection without response","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":787,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":534,"name":"_make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connection.py","lineno":565,"name":"getresponse"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":1430,"name":"getresponse"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":331,"name":"begin"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":300,"name":"_read_status"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-05T05:25:37.325035Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.326258Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.326404Z","level":"info","event":"Task:<Task(PythonOperator): create_custom_source_and_discover_schema>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-05T05:25:37.326534Z","level":"info","event":"Failure caused by ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","chan":"stdout","logger":"task"}
